src/Illuminate/Redis/Lua/LuaScript.php

```diff
@@ -0,0 +1,186 @@
+<?php
+
+namespace Illuminate\Redis\Lua;
+
+use Illuminate\Redis\Connections\Connection;
+use Illuminate\Redis\Lua\Exception\LuaScriptLoadException;
+
+/**
+ * The LuaScript class represents a Redis Lua script that can be executed by a Redis connection.
+ */
+class LuaScript
+{
+    /**
+     * The Redis connection instance.
+     *
+     * @var Connection
+     */
+    public readonly Connection $connection;
```

Comment:
We currently don't use `readonly` or typed properties in the framework for internal code. Please remove these.

Reply:
Thanks! will do